share/spice/seat_geek/events_by_city/seat_geek_events_by_city.css

```diff
@@ -0,0 +1,77 @@
+.zci--seat_geek .tile--seat_geek {
```

Comment:
@andrey-p All these references must be changed as well, to reflect the new ids, or the rules won't be applied:
![screen shot 2015-01-22 at 20 41 29](https://cloud.githubusercontent.com/assets/3652195/5863185/ceb28f8a-a277-11e4-8414-a9397727e110.png)